sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -864,6 +868,7 @@ SHARDS: 'SHARDS';
 PRIMARY_KEY: 'PRIMARY KEY';
 OFF: 'OFF';
 FULLTEXT: 'FULLTEXT';
+FILTER: 'FILTER';
```

Comment:
Can we add this to `nonReserved` ? Otherwise we've a breaking change due to a new keyword.